.ci/setup.sh

```diff
@@ -39,4 +51,20 @@ then
         --body-length 72 \
         --subject-length 75 \
         --verbose
+    # Travis doesn't provide a VT-x environment, so nothing more to do
+    # here.
+    exit 0
 fi
+
+# Check agent builds
+cd go get -d ${github_project} || true
+cd ${project_dir}
```

Comment:
Use `pushd` to be consistent with the rest of this script.
And don't forget `popd` at the end of this block.